### PR TITLE
Add support for angular 9

### PR DESCRIPTION
**Pull request by lehoffma**
I let the `ng update @angular/core @angular/cli` migration run on the whole project, which means the demo app now uses Angular 9 too.

### CODE REVIEW

**Comment by alexsukhodolsky** [comment]
@tnicola please merge this PR ... the current version is preventing anyone from upgrading to angular 10. Thanks!

File: angular.json [hunk 1]
```diff
@@ -52,7 +57,7 @@
                         "main": "projects/demo/src/main.ts",
                         "polyfills": "projects/demo/src/polyfills.ts",
                         "tsConfig": "projects/demo/tsconfig.app.json",
-                        "aot": false,
+                        "aot": true,
```
Comment (tnicola) [inline angular.json:60]: @alexsukhodolsky  I would be happy to merge it but unfortunately the unit test compilation is failing. Probably due to this change. @lehoffma Can you check this please?

Comment (lehoffma) [reply angular.json:60]: I tried that, but it didn't change the test results for me. The broken unit tests were mostly due to the updated libraries like rxjs I think. I rebased and pushed a fix for the tests, let me know if there's anything else I can do.

**Comment by tnicola** [comment]
@lehoffma PR merged, thanks!A new version of the package has been released: ngx-joyride@2.3.0

**Comment by alexsukhodolsky** [comment]
awesome--thanks @tnicola and @lehoffma!

**Pull request merged**